src/components/Table/GridTable.stories.tsx

```diff
@@ -159,7 +159,7 @@ export function NestedRows() {
     child: (row) => <CollapseToggle row={row} />,
     grandChild: () => "",
     add: () => "",
-    w: 0,
+    w: "60px",
```

Comment:
**question:** Since we are no longer using CSS Grid, are we losing out on responsiveness?

Reply:
I left some comments in Chromatic!

Reply:
Right, so I think we can be responsive to screen width (fr) but not like content-within-the-table (minmaxs, etc.). Right? 

And our rationale for not allowing responsive-to-content-within-the-table is that it makes cross-row column alignment hard i.e. have to use css-grid/table. And even in virtual tables, that is not enough b/c all content is not known at time of render.

Reply:
100% - well put. "response to the screen width, not the content within the table"